### DISCUSSION TITLE
2023 Ideas to contribute to the repo

### DISCUSSION THREAD

**Discussion by miguelgfierro** [recommenders-team/recommenders#2011] [Ideas]
- Scrapbook alternative https://github.com/recommenders-team/recommenders/issues/2012
- Python 3.10 https://github.com/recommenders-team/recommenders/issues/1919
- Support for TF>2.10.1 https://github.com/recommenders-team/recommenders/issues/2018
- External execution of tests https://github.com/recommenders-team/recommenders/issues/1853 
- SAR https://github.com/recommenders-team/recommenders/issues/1954
- VAE https://github.com/recommenders-team/recommenders/issues/1915
- P5 https://github.com/recommenders-team/recommenders/issues/1983
- Finish RippleNet PR https://github.com/recommenders-team/recommenders/pull/1014

**Comment by aidowu1** [comment]
Will it be possible to provide a notebook that can demonstrate how to use Azure ML's Responsible AI (RAI) to provide interpretability of the inference results of any of the recommendation models.

**Reply by miguelgfierro** [reply]
yes, that sounds like a good project
